pkg/agent/controller/noderoute/node_route_controller.go

```diff
@@ -204,15 +205,21 @@ func (c *Controller) removeStaleGatewayRoutes() error {
 	}
 
 	// TODO: This is not the best place to keep the ClusterIP Service routes.
-	desiredClusterIPSvcIPs := map[string]bool{}
+	desiredClusterIPSvcIPs := sets.NewString()
 	if c.proxyAll && runtime.IsWindowsPlatform() {
 		// The route for virtual IP -> antrea-gw0 should be always kept.
-		desiredClusterIPSvcIPs[config.VirtualServiceIPv4.String()] = true
+		vIPNet := util.NewIPNet(config.VirtualServiceIPv4).String()
+		desiredClusterIPSvcIPs.Insert(vIPNet)
```

Comment:
Let's make it clear this is ipNet not IP by changing the name to `desiredClusterIPNets`

Reply:
Done.